api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
It would probably fix this particular case... I propose, though, a longer check of the source code for other use cases. Perhaps, grep for the constant in the code.